FiveCalls/FiveCalls/IssueDone.swift

```diff
@@ -8,23 +8,161 @@
+        
+        if let titleString = try? AttributedString(markdown:  R.string.localizable.doneTitle(issue.name)) {
+            self.markdownTitle = titleString
+        } else {
+            self.markdownTitle = AttributedString(R.string.localizable.doneScreenTitle())
+        }
+    }
+        
+    let donateURL = URL(string: "https://secure.actblue.com/donate/5calls-donate?refcode=ios&refcode2=\(AnalyticsManager.shared.callerID)")!
+    var markdownTitle: AttributedString!
     
     var body: some View {
         ScrollView {
-            VStack {
-                Text("Issue \(issue.name) done page")
+            VStack(alignment: .leading) {
+                HStack {
+                    Spacer()
+                    Text(markdownTitle)
+                        .font(.title2)
+                    Spacer()
+                }.padding(.vertical, 16)
+                VStack {
+                    if let totalCalls = viewModel.totalCalls {
+                        CountingView(title: R.string.localizable.totalCalls(), count: totalCalls)
+                            .padding(.bottom, 14)
+                    }
+                    if let issueCalls = viewModel.issueCalls {
+                        CountingView(title: R.string.localizable.totalIssueCalls(), count: issueCalls)
+                            .padding(.bottom, 14)
+                    }
+                }.padding(.bottom, 16)
+                Text(R.string.localizable.support5calls())
+                    .font(.caption).fontWeight(.bold)
+                HStack {
+                    Text(R.string.localizable.support5callsSub())
+                    Button(action: {
+                        openURL(donateURL)
+                    }) {
+                        PrimaryButton(title: R.string.localizable.donateToday(), systemImageName: "hand.thumbsup.circle.fill", bgColor: .fivecallsRed)
+                    }
+                }.padding(.bottom, 16)
+                Text(R.string.localizable.shareThisTopic())
+                    .font(.caption).fontWeight(.bold)
+                ShareLink(item: issue.shareURL) {
+                    AsyncImage(url: issue.shareImageURL,
+                               content: { image in
+                        image.resizable()
+                            .aspectRatio(contentMode: .fill)
+                    }, placeholder: { EmptyView() })
+                }.padding(.bottom, 16)
                 Button(action: {
                     router.backToRoot()
                 }, label: {
-                    Text("Back to dashboard")
+                    PrimaryButton(title: R.string.localizable.doneScreenButton(), systemImageName: "flag.checkered")
+
                 })
             }
+            .padding(.horizontal)
         }.navigationBarHidden(true)
         .clipped()
+        .frame(maxWidth: 500)
+        .onAppear() {
+            loadStats()
+        }
+    }
+        
+    func loadStats() {
+        let operation = FetchStatsOperation()
```

Comment:
not required, but you could factor this out of the view using unidirectional flow allowing for better testability, maybe modifying the existing `FetchStats` to take an optional `issueID`?

Reply:
Looks like that might actually remove the need for the view model entirely as you aren't currently using `showDonate`. Also simplifies having both an `issue` model and `viewModel`

Reply:
Ugh, you're right. It's taking me a minute to make the unidirectional flow the default in my brain. Fixed